setup/db/db/schema-4920to41000.sql

```diff
@@ -62,3 +62,4 @@ INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`,
 INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`, `sort_order`) values (UUID(), 3, 'createSnapshotFromVMSnapshot', 'ALLOW', 302) ON DUPLICATE KEY UPDATE rule=rule;
 INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`, `sort_order`) values (UUID(), 4, 'createSnapshotFromVMSnapshot', 'ALLOW', 260) ON DUPLICATE KEY UPDATE rule=rule;
 
+ALTER TABLE `user_ip_address` ADD COLUMN `staticnat_state` VARCHAR(32) COMMENT 'static  rule state while removing'
```

Comment:
This column name does not match the name of the column in the VO (`rule_state`).